src/components/Common/UI/NumberInput/NumberInput.module.scss

```diff
@@ -37,28 +28,24 @@
     display: flex;
     width: 100%;
     border-radius: var(--g-radius);
-
-    &[data-focus-within] {
-      outline: var(--g-focus-borderWidth) solid var(--g-focus-color);
-      @include formFocusOutline;
-    }
+    overflow: visible;
   }
 
-  :global(.react-aria-Input) {
-    background: var(--g-input-background);
-    border: var(--g-input-borderWidth) solid var(--g-input-borderColor);
-    border-radius: var(--g-radius);
-    color: var(--g-input-textColor);
-    font-size: var(--g-input-fontSize);
-    padding: var(--g-input-paddingY) var(--g-input-paddingX);
-    height: var(--g-input-height);
-    outline: none;
-    flex: 1;
-
-    &[data-disabled] {
-      border-color: var(--g-input-disabled-border);
-      color: var(--g-input-disabled-color);
-      background-color: var(--g-input-disabled-bg);
-    }
-  }
+  // :global(.react-aria-Input) {
```

Comment:
Do we need to keep this?

Reply:
nope, was experimenting to make sure i could remove the overrides. will nix this!